### ISSUE TITLE
param `--wait-ready` not working in cluster_mode

### ISSUE THREAD

**Issue by yuzhigang33** [Unitech/pm2#2890]
**What's going wrong?**

When i run the example app `wait_ready`, it's not working. The app start online immediately.

**How could we reproduce this issue?**

```
cd examples/wait-ready
pm2 start ecosystem.json
```
## Supporting information

```
PM2 version: `2.4.6`
Node version: `7.10.0`
Windows? Mac? Linux?
MacOS Sierra 10.12.4
```

**Comment by yuzhigang33** [comment]
@vmarchaud 
Hi, does cluster_mode support '--wait-ready' option ?
I find this pr: https://github.com/Unitech/pm2/pull/2472 , but the code is missing in the latest version.

**Comment by vmarchaud** [comment]
@yuzhigang33 Don't why the example doesn't work but i'm sure the code is on the latest version

**Comment by evollu** [comment]
from the code cluster_mode doesn't support wait-ready
https://github.com/Unitech/pm2/blob/master/lib/God.js#L194

**Comment by vmarchaud** [comment]
Published under PM2 `2.5`
